cms/templates/group_configurations.html

```diff
@@ -33,31 +31,35 @@ <h1 class="page-header">
         <small class="subtitle">${_("Settings")}</small>
         <span class="sr">&gt; </span>${_("Group Configurations")}
       </h1>
-      <nav class="nav-actions">
-        <h3 class="sr">${_("Page Actions")}</h3>
-        <ul>
-          <li class="nav-item">
-            <a href="#" class="button new-button"><i class="icon-plus"></i> ${_("New Group Configuration")}</a>
-          </li>
-        </ul>
-      </nav>
     </header>
   </div>
 
   <div class="wrapper-content wrapper">
     <section class="content">
       <article class="content-primary" role="main">
-      % if configurations is None:
-        <div class="notice notice-incontext notice-moduledisabled">
-            <p class="copy">
-                ${_("This module is disabled at the moment.")}
-            </p>
+        <div class="wrapper-groups cohort-groups">
+          <h3 class="title">${_("Cohorted Content")}</h3>
+          <p class="copy">${_("A cohorted content group configuration allows different groups of students to view separate content. You can then define what units each cohort can see from the {a_start}Course Outline{a_end}. [Copy TBD]").format(a_start='<a href="{}">'.format(course_outline_url), a_end="</a>")}</p>
```

Comment:
This text should already have been deleted per outdated diff? Sidebar text sufficient.